workerd.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: workerd
+  version: 1.20230904.0
+  epoch: 0
+  description: The JavaScript / Wasm runtime that powers Cloudflare Workers
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - wolfi-baselayout
+      - binutils
+      - build-base
+      - git
+
+      - bazel-6
+      - openjdk-11
+
+      - bash
+      - gnutar
+
+      - python3
+      - clang-16
+      - clang-16-dev
+      - llvm16
+      - llvm16-dev
+      - llvm-lld-16
+
+      - llvm-libcxx-16-dev
+      - llvm-libcxx-16
+      # - llvm-libcxxabi-16
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cloudflare/workerd
+      tag: v${{package.version}}
+      expected-commit: b4b25357f47e737621098dde66fa0d0c6cb58450
+      destination: workerd
+
+  - runs: |
+      ln -s /var/cache/melange .cache
+      cd workerd
+      JAVA_HOME=/usr/lib/jvm/java-11-openjdk bazel sync --configure
```

Comment:
The `ln` and `bazel sync` are only useful for local development (to persist the bazel cache across builds) and should be deleted before merging.

Bazel inside of melange is a bit of a pain.